version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.2.7-SNAPSHOT"
+version in ThisBuild := "0.2.7"
```

Comment:
```suggestion
version in ThisBuild := "0.2.7-SNAPSHOT"
```

This file is not needed anymore. Please, accept my suggestion, take a look at this PR: https://github.com/47degrees/sbt-hood/pull/106 . Thanks.

Reply:
The file has been deleted already after #106 was merged 🙏